packages/bot-web-ui/src/stores/server-side-bot-store.ts

```diff
@@ -378,8 +378,15 @@ export default class ServerBotStore {
         try {
             const items = [[localize('Journal')]];
             [...this.journal].map(item => {
-                const arr = [item.msg];
-                items.push(arr);
+                let combinedMessage;
+                if (item?.type === JOURNAL_TYPE.WON) {
+                    combinedMessage = `Profit amount: ${item?.amount ?? ''}`;
+                } else if (item?.type === JOURNAL_TYPE.LOSS) {
+                    combinedMessage = `Loss amount: ${item?.amount ?? ''}`;
+                } else {
+                    combinedMessage = `${item?.type ?? ''}:${item?.msg ?? ''}`;
+                }
+                items.push([combinedMessage]);
```

Comment:
```suggestion
let combinedMessage;

switch (item?.type) {
    case JOURNAL_TYPE.WON:
        combinedMessage = `Profit amount: ${item?.amount ?? ''}`;
        break;
    case JOURNAL_TYPE.LOSS:
        combinedMessage = `Loss amount: ${item?.amount ?? ''}`;
        break;
    default:
        combinedMessage = `${item?.type ?? ''}:${item?.msg ?? ''}`;
        break;
}

items.push([combinedMessage]);
```

@mayuran-deriv can we go for switch case here since the checks are on the discrete value of JOURNAL_TYPE?